classes/workflow/EditorDecisionActionsManager.inc.php

```diff
@@ -164,33 +158,17 @@ static function _externalReviewStageDecisions() {
 				'operation' => 'sendReviewsInReview',
 				'name' => 'requestRevisions',
 				'title' => 'editor.submission.decision.requestRevisions',
-				'image' => 'revisions',
-				'help' => 'editor.review.NotifyAuthorRevisions',
-				'titleIcon' => 'revisions_required',
-			),
-			SUBMISSION_EDITOR_DECISION_RESUBMIT => array(
```

Comment:
Maybe to leave this -- it is a valid decision -- but to leave out the 'operation' index (or so) and to always check if  that 'operation' index exist i.e. for actions to be displayed, calculated here https://github.com/pkp/pkp-lib/blob/master/pages/workflow/PKPWorkflowHandler.inc.php#L293 (I am not sure if also somewhere else this was used) ?

Reply:
Ah now I see how it is used i.e. where everywhere... Sorry!

Reply:
I got an email with another comment saying this removal is ok, but I don't see that here. Did the comment get deleted? Is it still an issue?